### PR TITLE
Add Hyperf v1.0

### PR DESCRIPTION
**Pull request by huangzhhui**

### CODE REVIEW

**Comment by joanhey** [comment]
Add Hyperf to travis jobs. (PHP/hyperf)
https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/.travis.yml#L103

**Comment by joanhey** [comment]
Please try to delete extra files not necessary to run the bencmark:

- .gitlab-ci.yml
- .php_cs
- .phpstorm.meta.php
- deploy.test.yml
- phpstan.neon
- test directory
- ...

File: frameworks/PHP/hyperf/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# 介绍
```
Comment (joanhey) [inline frameworks/PHP/hyperf/README.md]: Please try to translate it to English.
Will be more useful to all.

File: frameworks/PHP/hyperf/benchmark_config.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                "update_url": "/updates/",
+                "plaintext_url": "/plaintext",
+                "port": 9501,
+                "approach": "Realistic",
+                "classification": "Fullstack",
+                "database": "MySQL",
+                "framework": "Hyperf",
+                "language": "PHP",
+                "flavor": "None",
+                "orm": "Full",
```
Comment (joanhey) [inline frameworks/PHP/hyperf/benchmark_config.json]: It isn't a full ORM test.
In the _queries_ and _update_ tests, it is using `->to_array()`, so it isn't mapping the rows to objects.

File: frameworks/PHP/hyperf/benchmark_config.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                "platform": "swoole",
+                "webserver": "swoole",
```
Comment (joanhey) [inline frameworks/PHP/hyperf/benchmark_config.json]: It is using the web server from the platform `swoole`. 
Will be "webserver": "None",

File: frameworks/PHP/hyperf/app/Controller/IndexController.php [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        $rows = Fortune::all();
+
+        $insert = new Fortune();
+        $insert->id = 0;
+        $insert->message = 'Additional fortune added at request time.';
+
+        $rows->add($insert);
+        $rows = $rows->sortBy('message');
+
+        $html = '<!DOCTYPE html><html><head><title>Fortunes</title></head><body><table><tr><th>id</th><th>message</th></tr>';
```
Comment (joanhey) [inline frameworks/PHP/hyperf/app/Controller/IndexController.php]: Please try to use a template engine in the fortunes test.
This isn't a realistic approach in a full stack framework.

Also the requeriments say:

- The sorted list must be provided to a server-side template and rendered to simple HTML (see below for minimum template). The resulting HTML table displays each Fortune's id number and message text.
- The HTML generated by the template must be sent as a response.
- The implementation is encouraged to use best practices for templates such as layout inheritence, separate header and footer files, and so on. However, this is not required. We request that implementations do not manage assets (JavaScript, CSS, images). We are deferring asset management until we can craft a more suitable test.

**Comment by huangzhhui** [comment]
@joanhey I have been modified all your suggests, please review again.

**Comment by joanhey** [comment]
Hi @huangzhhui 

Now travis-ci is testing the Hyperf framework.
But the dockefile is failing.
https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/571760446#L720-L747

In the dockerfile I will delete the `ENV TIMEZONE=${timezone:-"Asia/Shanghai"}` and his use. As the server by default use `America/Los_Angeles`, to avoid time problems in tests.

The changes are OK.

**Comment by huangzhhui** [comment]
@joanhey All tests are passed, please review again.

**Comment by huangzhhui** [comment]
@nbrady-techempower Could you review and merge the PR ?

**Comment by nbrady-techempower** [comment]
@huangzhhui Everything looks good but can you please remove some of the `config/autoload` files.

- commands.php
- devtool.php
- middlewares.php
- redis.php

I know at least these ca be removed, maybe other. Will merge after that, thanks!

**Comment by huangzhhui** [comment]
All useless files have been removed, please review again @nbrady-techempower

**Comment by nbrady-techempower** [comment]
Thanks @huangzhhui Will merge once travis passes.

**Pull request merged**